test/bigtest/tests/login-test.js

```diff
@@ -19,13 +27,355 @@ describe('Login', () => {
     expect(login.submit.isDisabled).to.be.true;
   });
 
-  describe('with invalid credentials', () => {
-    it.always('does not log in');
-    it('displays an error');
+  it('error message should not be present', () => {
+    expect(login.message.isPresent).to.be.false;
+  });
+
+  describe('username insertion', () => {
+    beforeEach(async () => {
+      const username = login.username;
+
+      await username.fill('test');
+    });
+
+    it('has an active submit button', () => {
+      expect(login.submit.isPresent).to.be.true;
+      expect(login.submit.isDisabled).to.be.false;
+    });
+
+    it('error message should not be present', () => {
```

Comment:
```suggestion
    it.always('error message should not be present', () => {
```

Reply:
In general, the same feedback applies everywhere we test that something `should not` happen. So I won't leave these comments all over the place.